x/employeestore/internal/types/types.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// MinNamePrice is Initial Starting Price for a name that was never previously owned
+var MinNamePrice = sdk.Coins{sdk.NewInt64Coin("nametoken", 1)}
+
+// Whois is a struct that contains all the metadata of a name
+type Whois struct {
```

Comment:
I see whois from namespace was used throughout application,we don't need an owner as per discussed usecase.